Library/Formula/jump.rb

```diff
@@ -0,0 +1,35 @@
+require "language/go"
+
+class Jump < Formula
+  desc "Quick and fuzzy directory jumper."
+  homepage "https://github.com/gsamokovarov/jump"
+  url "https://codeload.github.com/gsamokovarov/jump/tar.gz/v0.5.1"
+  sha256 "6546efc8e8c63bd64e2b2fd8ff785c80e43d094014765c55b94a0fc1e811c6f2"
+  head "https://github.com/gsamokovarov/jump.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/gsamokovarov").mkpath
+    ln_s buildpath, buildpath/"src/github.com/gsamokovarov/jump"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", "#{bin}/jump"
+    man1.install "man/jump.1"
+  end
+
+  def caveats; <<-EOS.undent
+    To install the shell integration, follow the instructions at:
+      #{bin}/jump shell
+    EOS
+  end
```

Comment:
I think unless they are Homebrew-specific instructions (which they aren't): better to just assume people will look at the homepage for this. If the "integration" is e.g. bash completion it'd be ideal to just install that as usual for people.